web_advanced_search/static/src/js/DomainSelector.esm.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import {patch} from "@web/core/utils/patch";
+import {_t} from "web.core";
+import Domain from "web.Domain";
+import DomainSelector from "web.DomainSelector";
+import basic_fields from "web.basic_fields";
+/**
+ * The redraw in the Debug Field does not trigger correctly
+ * so we overwrite it with the v14 Version
+ *
+ */
+patch(DomainSelector.prototype, "web.DomainSelector", {
+    /**
+     * @override
+     */
+    _onDebugInputChange(e) {
+        if (!$(".o_add_advanced_search").length) {
+            return this._super(...arguments);
+        }
+        const rawDomain = e.currentTarget.value;
+        try {
+            Domain.prototype.stringToArray(rawDomain);
+        } catch (err) {
+            // If there is a syntax error, just ignore the change
+            this.displayNotification({
+                title: _t("Syntax error"),
+                message: _t("Domain not properly formed"),
```

Comment:
For this to be exported for translation, we need to override http.py, isn't it?

Reply:
That's for frontend

Reply:
OK then